lib/vsc/install/ci.py

```diff
@@ -42,6 +42,7 @@
     vsc_setup,
     )
 
+RUFF_VERSION = "0.13.1"
```

Comment:
do you know if it makes sense to set this to latest? you print the version each time it runs, so if latest starts failing, we know the last working version to lock it to.
not sure how often it changes and if the changes are manageable with each release. i am not against letting this free.